templates/eap74-sso-s2i.json

```diff
@@ -49,6 +49,13 @@
             "value": "redhat-sso74-openshift:latest",
             "required": true
         },
+        {
+            "displayName": "Custom http Route Hostname",
+            "description": "Hostname for http service route (e.g. eap-app-myproject.example.com).  Required for SSO-enabled applications.  This is added to the white list of redirects in the SSO server.",
+            "name": "HOSTNAME_HTTP",
+            "value": "",
+            "required": true
```

Comment:
Perhaps it should stay `"required": false` for new parameter? I'm not sure about this

Reply:
+1 this should have a default value or not be required, I'll check how it used to get set.

Reply:
See here:

https://github.com/jboss-openshift/application-templates/blob/master/eap/eap71-amq-persistent-s2i.json#L33

As the comment says: "Custom hostname for http service route.  Leave blank for default hostname, e.g.: <application-name>-<project>.<default-domain-suffix>"

Reply:
I assume you refer to https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/launch/keycloak/added/keycloak.sh#L990

Do we want to change also `HOSTNAME_HTTPS`?

Reply:
No, sorry my comment got chopped off. If its blank in the template OpenShift will default the value to secure-application-name-project.default-domain-suffix. So it should always be set.

Reply:
updated